deployment/terraform/main.tf

```diff
@@ -102,6 +102,10 @@ module "modelservice" {
   # App
   db_endpoint = "${var.modelservice["db_endpoint"]}"
   graphite_url = "${var.modelservice["graphite_url"]}"
+  detector_mapper_es_urls = "${var.modelservice["detector_mapper_es_urls"]}"
+  detector_mapper_es_config_vars_json = "${var.modelservice["detector_mapper_es_config_vars_json"]}"
+  detector_mapper_index_name = "{var.modelservice["detector_mapper_index_name"]}"
+  detector_mapper_doctype = "{var.modelservice["detector_mapper_doctype"]}"
```

Comment:
Lines 107 and 108 are missing leading `$`.

Reply:
Fixed.